src/app/static/src/app/store/translations/locales.ts

```diff
@@ -533,7 +533,7 @@ const en: Translations = {
     noteHeader: "Notes: (your reason for saving as a new version)",
     notUsed: "Not used",
     ok: "OK",
-    or: "or",
+    or: " or",
```

Comment:
makes sure padding between create new project or return to current project on projects page